brain/circuit.h

```diff
@@ -61,10 +61,8 @@ class Circuit : public boost::noncopyable
 
     /**
      * @return The set of GIDs for the given target name. If empty it will
```

Comment:
I don't get what you mean by this. Are you saying that getGIDs() should take no arguments? If that's the case I don't agree with you at all. As a user I'd be very annoyed by having to deal with all the code for creating a blue config, then the target parsers based on the information provided by the blue config, and then call brion::Target::parse. If getGIDs() takes no arguments all the code in brain::Circuit that is already simplifying creeps from where it's nicely hidden into user code.